common/utils/iiif.ts

```diff
@@ -276,3 +277,31 @@ export function getSearchService(manifest: IIIFManifest): Service | undefined {
     return manifest.service;
   }
 }
+
+export function getThumbnailService(
+  canvas: IIIFCanvas
+): IIIFThumbnailService | undefined {
+  const service = canvas?.thumbnail?.service;
+  if (Array.isArray(service)) {
+    return service[0];
+  } else {
+    return service;
+  }
+}
```

Comment:
Do we do this because the new IIIF is different?

Do we have a way of knowing how to get rid of checks the become redundant once we swap over?

If we moved this information to the hook, we could have something like:
```ts
if (toggle) getIIIFThumbnailService
if (!toggle) getIIIFV2ThumbnailService
```

Making it really clear as to what we should remove once we've swapped over.

Reply:
> Do we do this because the new IIIF is different?

yes, [but it won't be long term](https://wellcome.slack.com/archives/CBT40CMKQ/p1618389639350900?thread_ts=1618218978.347700&cid=CBT40CMKQ
)

The trouble with moving it to the hook is that the Viewer components aren't using the hook and it becomes a much bigger refactor, which I'm not sure is worth it at this point. I don't think it's the end of the world if it remains - we'll be moving to iiif V3 with all the proper types available and these function will be updated then

Reply:
> I don't think it's the end of the world

We need to be keeping an eye on tech debt as we go on as we don't have tech cleanup sprints, and don't want them.

Could we at least add a comment in the `getThumbnailService` so that if someone comes up against it in the future the can clean it up?

Reply:
To maybe rephrase myself, it's not the end off the world, but can be an annoying experience for the next developer who comes to this code and has to unpick why it is the way it is, only to conclude it doesn't need to be this way.